cpp/src/io/utilities/column_buffer.hpp

```diff
@@ -86,7 +81,7 @@ struct column_buffer {
                 bool _is_nullable                   = true,
                 rmm::cuda_stream_view stream        = rmm::cuda_stream_default,
                 rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource())
-    : type(_type), is_nullable(_is_nullable), _null_count(0)
+    : _null_count(0), type(_type), is_nullable(_is_nullable)
```

Comment:
_null_count is already set to zero in the inline initializer, we can remove it here.
```suggestion
    : type(_type), is_nullable(_is_nullable)
```